lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -66,10 +66,18 @@ class AwsCompileFunctions {
   compileFunction(functionName) {
     const newFunction = this.cfLambdaFunctionTemplate();
     const functionObject = this.serverless.service.getFunction(functionName);
+    let artifactFilePath;
```

Comment:
It seems like a unit test for all the changes here is missing (in `index.test.js`)